tests/e2e/scenarios/bare-metal-testdriver.go

```diff
@@ -84,7 +84,8 @@ func initBaremetalDriver() *baremetalDriver {
 
 var _ storageframework.TestDriver = &baremetalDriver{}
 var _ storageframework.DynamicPVTestDriver = &baremetalDriver{}
-var _ storageframework.EphemeralTestDriver = &baremetalDriver{}
+// todo add test coverage for generic ephemeral volumes
```

Comment:
Maybe skip simple [EphemeralVolume ](https://github.com/kubernetes/kubernetes/blob/7f5380a4035edb083cf163647158a05b12057157/test/e2e/storage/framework/testpattern.go#L130) testpattern [here ](https://github.com/dell/csi-baremetal/blob/f5e629aa544f90310f3ca39455be3442e72c77d4/tests/e2e/scenarios/bare-metal-testdriver.go#L125). Generic Ephemeral Volumes should become supported automatically

Reply:
Thanks, will try

Reply:
Done